Sources/ApolloCodegenLib/ApolloSchemaDownloader.swift

```diff
@@ -29,7 +31,11 @@ public struct ApolloSchemaDownloader {
       case .couldNotCreateSDLDataToWrite(let schema):
         return "Could not convert SDL schema into data to write to the filesystem. Schema: \(schema)"
       case .couldNotConvertIntrospectionJSONToSDL(let underlying):
-          return "Could not convert downloaded introspection JSON into SDL format. Underlying error: \(underlying)"
+        return "Could not convert downloaded introspection JSON into SDL format. Underlying error: \(underlying)"
+      case .couldNotCreateURLComponentsFromEndpointURL:
+        return "Could not create URLComponents from EndpointURL for Introspection."
+      case .couldNotGetURLFromURLComponents:
+        return "Could not get URL from URLComponents."
```

Comment:
Might be worth passing the components here so people can examine them. Also fine not to.